snippets/mssql.json

```diff
@@ -294,5 +294,54 @@
 			"GO"
 		],
 		"description": "Get Space Used by Tables"
+	},
+
+	"Lists all procedures by schema": {
+		"prefix": "sqlListProcedures",
+		"body": [
+			"-- Lists all procedures by schema ${1:ShemaName}",
+            "SELECT",
+            "SchemaName = s.name,",
+            "ProcedureName = pr.name,",
+            "ProcId = pr.object_id",
+            "FROM",
+            "sys.procedures pr",
+            "INNER JOIN",
+            "sys.schemas s ON pr.schema_id = s.schema_id",
+            "WHERE",
+            "s.name = '${1:ShemaName}'",
+			"GO"
+		],
+		"description": "Lists all procedures by schema"
+	},
+
+
+	"Makes Header Documentation for SQL Stored Procedure": {
+		"prefix": "sqlDocProcedure",
+		"body": [
+            "--",
+            "-- STORED PROCEDURE",
+            "--     ${1:ProcedureName}",
```

Comment:
Please use `\t` instead of whitespace. VSCode handles converting tab to spaces depending on user preference, but if you use spaces people who prefer tabs will be forever unhappy.